aws/eks-values.yaml

```diff
@@ -28,6 +30,26 @@ securityContext:
 env:
   AWS_MARKETPLACE_PRODUCT_CODE: 4qd427bqyuzyhn5st1v7x40oq
 
+livenessProbe:
+  enabled: true
+  # You may want to increase the initialDelaySeconds for the bigger LLMs as Llama 3.1 405B
+  # since the download will take longer and the default delay may not be enough for the
+  # download to be completed
+  # initialDelaySeconds: 360
+  # periodSeconds: 15
+  # timeoutSeconds: 5
+  # failureThreshold: 3
+
+readinessProbe:
+  enabled: true
+  # You may want to increase the initialDelaySeconds for the bigger LLMs as Llama 3.1 405B
+  # since the download will take longer and the default delay may not be enough for the
+  # download to be completed
+  # initialDelaySeconds: 360
+  # periodSeconds: 30
+  # timeoutSeconds: 5
+  # failureThreshold: 3
+
```

Comment:
I initially removed those, but when serving Meta Llama 3.1 405B the defaults will fail as the download will take longer than the `readinessProbe` and `livenessProbe` for the `initialDelaySeconds`, so it will fail, and if the defaults are enforced then the user won't be able to modify those and some deployments may always just fail due to the probes.